src/main/java/sg/ncl/MainController.java

```diff
@@ -823,9 +829,10 @@ public String validateDetails(
 
                 teamFields.put("name", signUpMergedForm.getTeamName().trim());
                 teamFields.put("description", signUpMergedForm.getTeamDescription().trim());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'DESCRIPTION' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)